### PR TITLE
 create testtool build

### PR DESCRIPTION
**Pull request by wengangJi**
The script is started with a container project for the main project and test project based on the github account of the fork and is used for project testing. The boot parameter provides the global configuration of yaml through the docker project.
To do this, execute the build.sh file, enter the fork user name according to the prompt, and then start executing and starting the project.
The output result is expected to start the docker container for the main project and test project, and start the corresponding database service container according to the environment variable.
After startup, you can access the UI page at http://localhost:8080.

### CODE REVIEW

File: docker-compose-es.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+version: '3'
+services:
+  skywalking-es:
+    container_name: skywalking-es
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.8
```
Comment (peng-yongsheng) [inline docker-compose-es.yaml]: What's this? database docker image definition must in the docker project.

File: docker-compose-h2.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#    networks:
+#      - skywalking
+#    ports:
+#      - "8080:8080"
+#      - "10800:10800"
+#      - "11800:11800"
+#      - "12800:12800"
+    command: ["bash","collectorService.sh"]
+
+  skywalking-webappService:
```
Comment (peng-yongsheng) [inline docker-compose-h2.yaml]: Did not need web project for auto test.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+version: '3'
```
Comment (peng-yongsheng) [inline docker-compose.yaml]: What's the usage about this file? Same as `docker-compose-es.yaml`.

**Review by peng-yongsheng — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 3 above)
Comment (wengangJi) [reply docker-compose.yaml]: This is a legacy file that needs to be cleared. I'll delete it.

File: docker-compose-h2.yaml (hunk 2 above)
Comment (wengangJi) [reply docker-compose-h2.yaml]: I will be delete it.

File: docker-compose-es.yaml (hunk 1 above)
Comment (wengangJi) [reply docker-compose-es.yaml]: Docker compose have involved in the configuration file path to the mount, and according to the environment variables to pursue to the configuration file path, so currently only in the main project. If you're going to be in docker engineering, I need to rebuild implementation.

**Pull request merged**